resources/ts/Components/Integrations/Detail/CredentialsAuth0Clients.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useMemo } from "react";
+import { Heading } from "../../Heading";
+import { useTranslation } from "react-i18next";
+import {
+  Auth0Client,
+  Integration,
+  UiTiDv1Consumer,
+} from "../../../Pages/Integrations/Index";
+import { StatusLight } from "../../StatusLight";
+//import { ButtonPrimary } from "../../ButtonPrimary";
+import { CopyText } from "../../CopyText";
+import { ActivationFlow } from "../../ActivationFlow";
+import { OpenWidgetBuilderButton } from "../../IntegrationCard";
+import { IntegrationType } from "../../../types/IntegrationType";
+import { IntegrationStatus } from "../../../types/IntegrationStatus";
+
+type Props = {
+  auth0Clients: Auth0Client[];
+  uiTiDv1Consumers: UiTiDv1Consumer[];
+  email: string;
+  id: Integration["id"];
+  status: Integration["status"];
+  subscription: Integration["subscription"];
+  type: Integration["type"];
+};
+
+export const CredentialsAuth0Clients = ({
+  auth0Clients,
+  id,
+  status,
+  email,
+  subscription,
+  type,
+}: Props) => {
+  const { t } = useTranslation();
+
+  const auth0TestClient = useMemo(
+    () => auth0Clients.find((client) => client.tenant === "test"),
+    [auth0Clients]
+  );
+  const auth0ProdClient = useMemo(
+    () => auth0Clients.find((client) => client.tenant === "prod"),
+    [auth0Clients]
+  );
+
+  const auth0TestClientWithLabels = [
+    {
+      label: "details.credentials.client_id",
+      value: auth0TestClient?.clientId,
+    },
+    {
+      label: "details.credentials.client_secret",
+      value: auth0TestClient?.clientSecret,
+    },
+  ];
+
+  const auth0ProdClientWithLabels = [
+    {
+      label: "details.credentials.client_id",
+      value: auth0ProdClient?.clientId,
+    },
+    {
+      label: "details.credentials.client_secret",
+      value: auth0ProdClient?.clientSecret,
+    },
+  ];
+
+  return (
+    <>
+      <Heading className="font-semibold lg:min-w-60" level={4}>
+        {t("details.credentials.uitid_v2")}
+      </Heading>
+      {/*{!auth0ProdClient ? (
```

Comment:
Are you leaving this code, for when there is more clarity regarding the visibility of credentials?

Reply:
Yes